components/automate-deployment/pkg/majorupgradechecklist/v4.go

```diff
@@ -110,6 +111,8 @@ This is because automate version 4 now only supports this format due to AWS SDK
 	msg            = "\nFollow the guide below to learn more about reindexing:\nhttps://www.elastic.co/guide/en/elasticsearch/reference/6.8/docs-reindex.html"
 	oldIndexError  = "The index %s is from an older version of elasticsearch version %s.\nPlease reindex in elasticsearch 6. %s\n%s"
 	indexBatchSize = 10
+
+	maintenanceModeMsg = "To Upgrade to version 4.x we will have to put the system in maintenance mode. While the system is in maintenance mode 'ON' no new ingestion of data can happen. \nAt the end of successful upgrade, the maintenance mode will be switched off automatically and at end of un-successful upgrade, you have to set it ‘Off’ manually.\nAre you ready to proceed? "
```

Comment:
This upgrade put the system in maintenance mode. During that period no new ingestion of data can happen. \n The maintenance mode will be switched off automatically at the end of a successful upgrade.  But in case of an unsuccessful upgrade, you have to set it ‘Off’ manually.\nAre you ready to proceed?